cmd/keytransparency-server/main.go

```diff
@@ -206,7 +210,11 @@ func main() {
 		grpc.StreamInterceptor(grpc_prometheus.StreamServerInterceptor),
 		grpc.UnaryInterceptor(grpc_prometheus.UnaryServerInterceptor),
 	)
+
+	// TODO(ismail): replace with a non-core version (cmutations)?
```

Comment:
This is fine. We don't need to replace this with non-core version since this file is not imported.